index.html

```diff
@@ -303,6 +303,79 @@ <h4>+</h4>
           </div>
       </div>
     </div>
+        
+    <div class="sponsors" style="background-color: grey;">
+      <div class="container" style="padding: 3%; padding-bottom: 10%">
+          <div class="row">
+            <div class="col-sm">
```

Comment:
nit: i dont think we need to add a row/col for this header - same goes for the FAQ header